abci/types/types.go

```diff
@@ -112,22 +112,22 @@ func RegisterABCIServer(s grpc.Server, srv ABCIServer) {
 type CheckTxType = v1.CheckTxType
 
 const (
-	CheckTxType_New     CheckTxType = v1.CheckTxType_New
-	CheckTxType_Recheck CheckTxType = v1.CheckTxType_Recheck
+	CHECK_TX_TYPE_NEW     CheckTxType = v1.CHECK_TX_TYPE_NEW
```

Comment:
As discussed in #736, this cannot be renamed to `CHECK_TX_TYPE_UNSPECIFIED` due to prior usage. This means the enum cannot be extended in the future, as the newly added values will be mapped to 0 by protobuf bindings that have not been updated.